site/tests/universal/blockpage.test.ts

```diff
@@ -0,0 +1,52 @@
+import { expect, test } from "playwright-test-coverage";
+
+const codeBlockUrl = "/@hash/blocks/code";
+
+test("Updating block properties should update block preview", async ({
+  page,
+  isMobile,
+}) => {
+  await page.goto("/hub");
+
+  await expect(
+    page.locator('[data-testid="block-card"]', {
+      hasText: "Code",
+    }),
+  ).toHaveAttribute("href", codeBlockUrl);
+
+  await page.goto(codeBlockUrl);
+
+  await expect(page.locator("iframe[title='block']")).toBeVisible();
+
+  const blockFrameLocator = page.frameLocator("iframe[title='block']");
+
+  // wait till block is rendered
+  // @todo figure out why this doesn't come up on safari
```

Comment:
🤔 Please share the result when you find it out!

Reply:
I increased the timeout to 10000 for this to work on safari and iphone. Although I feel this behaviour is flaky and we'd probably have to watch and see if it fails in subsequent PRs. 

I noticed a different issue when trying to update the block properties via the editor which is peculiar to safari and iphone. Apparently parsing, updating and stringifying the json editor's value always results in an unescaped string in safari. 

![Screenshot 2022-07-22 at 03 02 20](https://user-images.githubusercontent.com/16928688/180346686-966bf092-e11f-4141-b144-32dbd4653ae8.png)

Reply:
Created an  [internal task](https://app.asana.com/0/1202542409311090/1202651551651719) to address this